### ISSUE TITLE
Crossover sections, redstone not removable when moving circuit

### ISSUE THREAD

**Issue by simpson409** [amadornes/SCM-Localization-Issues#208]
when i move the circuit with the cable on or remove the ones around it i cant remove the redstone in the crossover sections.
![2016-11-13_00 11 11](https://cloud.githubusercontent.com/assets/21246918/20242627/d209f9ea-a935-11e6-9835-89971667e99f.png)
![2016-11-13_00 11 25](https://cloud.githubusercontent.com/assets/21246918/20242628/d20db22e-a935-11e6-8582-f7756bb7c542.png)

**Comment by amadornes** [comment]
Yep. This is known and has been a thing for a very long time. I'm still not 100% sure how I could address it (apart from making the components drop, which would be annoying), so any input is welcome

**Comment by Inari-Whitebear** [comment]
Well at the very least just let people remove it like normal I guess.
